src/commonmark/en/resources/css/mkdocs_dhis2.css

```diff
@@ -280,12 +280,17 @@ article {
   color: var(--grey600);
 }
 
+.md-nav__link.md-nav__link--active {
+  color: var(--blue700);
```

Comment:
I'd also add
`background: var(--blue100)` and `border-radius: 3px;` here to make it even clearer that it is selected.

Reply:
This also looks better if you add `padding: 2px 0;` to `.md-nav__link` class as well.